prow/tide/status.go

```diff
@@ -251,6 +266,30 @@ func expectedStatus(queryMap *config.QueryMap, pr *PullRequest, pool map[string]
 		}
 		return github.StatusPending, fmt.Sprintf(statusNotInPool, minDiff)
 	}
+
+	indexKey := indexKeyPassingJobs(org, repo, baseSHA, string(pr.HeadRefOID))
+	passingUpToDatePJs := &prowapi.ProwJobList{}
+	if err := sc.pjClient.List(
+		context.Background(),
+		passingUpToDatePJs,
+		ctrlruntimeclient.MatchingField(indexNamePassingJobs, indexKey),
+	); err != nil {
```

Comment:
I like it especially for multiline statements, because the error handling nicely frames the statement, as long as the closing brace isn't put on the same line as the last argument.

Not going to insist on that thought, if its important to you

Reply:
Who else would you cite that follows this pattern?

I guess I'm confused that you don't consider
```golang
if err := foo(apples, oranges, pears); err != nil {
  // something
}
```
easier to follow than:
```golang
if err := foo(
  apples,
  oranges,
  pears,
); err != nil {
  // something
}
```

This seems to violate https://github.com/golang/go/wiki/CodeReviewComments#line-length

Reply:
No, I was comparing
```
err := foo(
  bar,
  baz)
```

to

```
if err := foo(
  bar,
  baz,
); err != nil { handleError }
```

Anyways I've changed this and will get myself a super-wide-x2 screen so it doesn't wrap when I have to of these side by side